01_fib/pfannerj/main.go

```diff
@@ -0,0 +1,37 @@
+package main
+
+import (
+	"fmt"
+	"io"
+	"os"
+)
+
+var fibout io.Writer = os.Stdout
+var mainout io.Writer = os.Stdout
+
+func main() {
+	fmt.Fprintf(mainout, "Fibonacci series starting...\n")
```

Comment:
Nit: this output was not requested in lab descriptions.

Reply:
Noted... leaving it for now as a possible value-add??